### PR TITLE
Refs [TASK][34922] Create UI Detail movie screen

### PR DESCRIPTION
**Pull request by manhhung23399**
## Related Tickets
- [#Task 34922: Content](https://edu-redmine.sun-asterisk.vn/issues/34922)
## WHAT
- Create ImageView to show poster movie
- Create TextView to show content movie, release date, score
- Create Chip to show genres
- Create Back button
- Create Button favorite
- Create RecyclerView Cast
- Create RecyclerView Companies
- Create RecyclerView Recommend movie
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_movie_detail.xml [hunk 1]
```diff
@@ -1,6 +1,246 @@
...
+            android:scaleType="centerCrop"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+            android:id="@+id/imageBackDetail"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dp_10"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_movie_detail.xml:26]: em nên thêm padding cho các button để người dùng người ta dễ bấm hơn 
padding = 16dp

File: app/src/main/res/layout/fragment_movie_detail.xml [hunk 2]
```diff
@@ -1,6 +1,246 @@
...
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/imageBackdropDetail"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_180"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_movie_detail.xml]: Em nên set chiều cao của View theo tỉ lệ màn hình hơn là fix cứng theo dp 
sử dụng height_percent

File: app/src/main/res/layout/fragment_movie_detail.xml [hunk 3]
```diff
@@ -1,6 +1,246 @@
...
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dp_10"
+            android:src="@drawable/ic_back"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/contraintDetail"
+            android:layout_width="match_parent"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_movie_detail.xml]: constraint layout, view con thì dùng match_constraint(0dp) thay vì match_parent

File: app/src/main/res/layout/fragment_movie_detail.xml [hunk 4]
```diff
@@ -1,6 +1,275 @@
...
+                android:text="@string/text_release"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/sp_18"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHeight_percent="0.1"
+                app:layout_constraintStart_toStartOf="@id/textDate"
+                app:layout_constraintTop_toBottomOf="@+id/textDate" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (KhacTung-Shjn) [inline app/src/main/res/layout/fragment_movie_detail.xml]: Dùng constrain group thử xem, sao lại lồng nhau như này

File: app/src/main/res/layout/item_movie_recommend.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (KhacTung-Shjn) [inline app/src/main/res/layout/item_movie_recommend.xml]: mấy file item này giống nhau liệu có dùng chung được không ?

**Pull request merged**